Unofficial.md

```diff
@@ -30,7 +30,7 @@ Unofficial positions for life may be proposed and awarded to members, subject to
 
 ## Presidential Duties
 
-1. Once elected the new president must cut a small lock of hair from the outgoing president and then keep the lock of hair on their person for their entire term as president, preferably in the form of a locket. This will serve as a reminder of the great traditions and responsibilities they have sworn to uphold. At the end of the term, at the following AGM, the lock of hair should be ceremonially burnt. At any point during their term any UBES Member or UBES Alumni can challenge the president to see the hair and if the President is not able to present it immediately then a forfeit will be applicable.
```

Comment:
I find it strange that the unofficial constitution refers to an official locket that is not legislated in any of the official regulations. As this is a constitution (and as such does not automatically expire), it would be necessary for the locket to be defined in the official constitution (or higher). As this has not been done (and is not desirable), please don’t refer to it as official here. Simply “a locket” suffices. This also does not put onus on the society to purchase a locket, which would not be justifiable as the unofficial constitution is not relevant for committee decision-making.

Reply:
Would you accept "unoffical locket"? :P

"a locket" is acceptable

Reply:
I'd just say “a locket”